README.md

```diff
@@ -1,2 +1,91 @@
 # test-automation
-Tools for making our tests easier to run
+
```

Comment:
I checked in most of these. Still need to update the README with better instructions but it might be ready for another look.